src/test/java/org/mule/extension/db/integration/function/oracle/CreateStructOracleTestCase.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.db.integration.function.oracle;
+
+import static java.lang.reflect.Array.get;
+import static java.util.Collections.emptyList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import org.mule.extension.db.integration.AbstractDbIntegrationTestCase;
+import org.mule.extension.db.integration.TestDbConfig;
+import org.mule.extension.db.integration.model.OracleTestDatabase;
+import org.mule.runtime.core.api.event.CoreEvent;
+
+import java.sql.SQLException;
+import java.sql.Struct;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runners.Parameterized;
+
+public class CreateStructOracleTestCase extends AbstractDbIntegrationTestCase {
+
+  @Before
+  public void init() throws SQLException {
+    ((OracleTestDatabase) this.testDatabase).createPersonType(getDefaultDataSource().getConnection());
+  }
+
+  @Parameterized.Parameter(4)
```

Comment:
organize this so that the parameters are on the top of the class before the methods.